spec/api/grape_api/v1/users_spec.rb

```diff
@@ -214,10 +214,10 @@
     end
   end
 
-  it "DELETE /api/v1/users/:id/application_tokens/:id returns status 405" do
+  it "DELETE /api/v1/users/:id/application_tokens/:id returns status 404" do
     user = create :user
     token = create :application_token, user: user
     delete "/api/v1/users/#{user.id}/application_tokens/#{token.id}", nil, @header
-    expect(response).to have_http_status(:method_not_allowed)
+    expect(response).to have_http_status(:not_found)
```

Comment:
I think that this should still be `405` because the resource exists and it's only the method that it's not allowed for `DELETE`. Not sure if it's easy to fix.

Reply:
As mentioned by you on private (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405), this change makes sense. 👍 

Thanks for the explanation! :)